### DISCUSSION TITLE
<select class="netbox-api-select"

### DISCUSSION THREAD

**Discussion by k3x** [netbox-community/netbox#12000] [Help Wanted!]
### Change Type

Addition

### Area

Development

### Proposed Changes

Hello,
i am writing a plugin for netbox. I am using a select element with the class netbox-api-select and the attribute data-url set.
Unfortunately i can not find any documentation about this.
Furthermore i want to change the data-url via javascript an reload the options. Is that possible? How?
Thanks!

```
<select class="netbox-api-select" data-url="/api/ipam/services/?brief=true" style="width:100%" id="choosenservice" ></select>
```

**Comment by kkthxbye-code** [comment]
This does not constitute a proper issue, so I'll convert it to a discussion.
